plutus-core/plutus-ir/src/PlutusIR/Transform/Inline.hs

```diff
@@ -152,48 +158,60 @@ type InliningConstraints tyname name uni fun =
     , PLC.ToBuiltinMeaning uni fun
     )
 
-
+-- | Useful info for inlining.
```

Comment:
I think this is actually so brief it's not worth it. A comment on a thing called `foo` that says "the foo" _actually_ doesn't add anything.